CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 * Allow the exception map in `Client` to be overridden. This is done in preparation of the upcoming CMA SDK.
+* The third parameter `$options` of the `Client::request()` method now accepts an optional value with key `baseUri`, which can be used to override the `Client`'s default internal value. Because of this, `Client::buildRequest()` now accepts an extra `$baseUri` optional parameter.
```

Comment:
I'd write a less detailed changeling entry for this one as (hopefully) no user will ever interact with this method directly. You should also mention that this is prep work for the CMA.